src/Label.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components'
+import { space, fontSize, color } from 'styled-system'
+
+const Label = styled.label`
+  font-size: 10px;
```

Comment:
If you're using the `fontSize` function, you should probably set this as a `defaultProp`